src/graphql/CollectionsProductsFragment.graphql

```diff
@@ -1,5 +1,5 @@
 fragment CollectionsProductsFragment on Collection {
-  products(first: $productsFirst) {
+  products(first: 20) {
```

Comment:
we want to hardcode this value?

Reply:
fragments have no access to variables.

Reply:
This wasn't initially valid, it just passed the parser somehow.

I'm not sure how many layers deep we want to parameterize things. Ideally, if people want heavy customization, people should be using the storefront API directly? But this is totally just opinion.

Reply:
is there a default that shopify uses across other places? if not, 20 seems reasonable to me

Reply:
There is on some of our APIs. The point of the change here was to get stuff to compile.

I can open these params up later by taking this out of a fragment.